DatadogRUM/Sources/DataModels/RUMDataModels.swift

```diff
@@ -3972,4 +4001,4 @@ public enum RUMMethod: String, Codable {
     case connect = "CONNECT"
 }
 
-// Generated from https://github.com/DataDog/rum-events-format/tree/78f17559b7898dad5a6b3b4af2fe4ab4a5be6b54
+// Generated from https://github.com/DataDog/rum-events-format/tree/3a5e8766c3a7a136607540e0f78d7810acf03038
```

Comment:
It points to feature branch from https://github.com/DataDog/rum-events-format/pull/188. This will be changed after merge in `rum-events-format` and before merge to `dd-sdk-ios/develop`.